async_lru/__init__.py

```diff
@@ -56,6 +56,8 @@ class _CacheParameters(TypedDict):
 class _CacheItem(Generic[_R]):
     fut: "asyncio.Future[_R]"
```

Comment:
Thinking further on this... if the _CacheItem now holds a reference to the task itself, does it even need a separate Future for the result?